### ISSUE TITLE
nuage.repo: Add example for bypassing proxy for local repo

### ISSUE THREAD

**Issue by jbemmel** [nuagenetworks/nuage-ospdirector#93]
In case a proxy is specified but some or all of the repos are local, it is possible to bypass the proxy (per repo) by doing the following:

[nuage]
name=nuage
baseurl=http://<url>
proxy=_none_
enabled=1
gpgcheck=1

**Comment by sunny-verma** [comment]
I agree its good to have but not all customer is behind a proxy but adding it just as note would be a good idea. I will add it in the next update to the documentation.
